ts/Dashboards/Plugins/HighchartsComponent.ts

```diff
@@ -751,30 +735,29 @@ class HighchartsComponent extends Component {
      *
      */
     private createChart(): Chart {
-        const charter = (
-            HighchartsComponent.charter ||
-            Globals.win.Highcharts as H
-        );
-        if (this.chartConstructor !== 'chart') {
-            const factory = charter[this.chartConstructor];
-            if (factory) {
-                try {
-                    return factory(this.chartContainer, this.chartOptions);
-                } catch {
-                    error(
-                        'The Highcharts component is misconfigured: `' +
-                        this.cell.id + '`'
-                    );
-                }
+        const charter = HighchartsComponent.charter || Globals.win.Highcharts;
+
+        if (!this.chartConstructor) {
+            this.chartConstructor = 'chart';
+        }
+
+        const factory = charter[this.chartConstructor] as any;
+        if (factory) {
+            try {
+                return factory(this.chartContainer, this.chartOptions);
+            } catch {
+                throw new Error(
+                    'The Highcharts component is misconfigured: `' +
+                    this.cell.id + '`'
+                );
             }
         }
 
         if (typeof charter.chart !== 'function') {
             throw new Error('Chart constructor not found');
         }
 
-        this.chart = charter.chart(this.chartContainer, this.chartOptions);
-        return this.chart;
+        return this.chart as any;
```

Comment:
I do not like the use of `any`. The previous one worked without it. Does it lack declarations in HighchartsTypes?